util/Migrator/DbScripts/2021-04-28_00_OrgResetPasswordAbilityAndRsaKeys.sql

```diff
@@ -12,12 +12,17 @@ UPDATE
     [dbo].[Organization]
 SET
     [UseResetPassword] = (CASE WHEN [PlanType] = 10 OR [PlanType] = 11 THEN 1 ELSE 0 END)
+WHERE
+    [UseResetPassword] IS NULL
 GO
 
-ALTER TABLE
-    [dbo].[Organization]
-ALTER COLUMN
-    [UseResetPassword] BIT NOT NULL
+IF COL_LENGTH('[dbo].[Organization]', 'UseResetPassword') IS NOT NULL
```

Comment:
You can remove this if check, the script would fail if the column didn't exist, so this is not necessary here; maybe to check the data type, but still shouldn't be necessary.